models/repo_list_test.go

```diff
@@ -60,10 +55,66 @@ func TestSearchRepositoryByName(t *testing.T) {
 		Page:     1,
 		PageSize: 10,
 		Private:  true,
-		Searcher: &User{ID: 14},
 	})
 
-	assert.NotNil(t, repos)
 	assert.NoError(t, err)
 	assert.Equal(t, int64(3), count)
+	assert.Len(t, repos, 3)
+
+	testCases := []struct {
+		name  string
+		opts  *SearchRepoOptions
+		count int
+	}{
+		{name: "PublicRepositoriesByName",
+			opts:  &SearchRepoOptions{Keyword: "big_test_", PageSize: 10},
```

Comment:
You should add similar checks as in api_repo_tests for all test cases. E.g. repo name, private/public repositories

Reply:
Done